### PR TITLE
Import speakers

### PR DESCRIPTION
**Pull request by jebbinSCL**

### CODE REVIEW

File: TrelloMigrate/SrmImport.fs [hunk 1]
```diff
@@ -6,11 +6,20 @@ open SrmApiClient
...
+let private importSpeakers (wrapper : SrmWrapper) = 
+    let speakerIsAdmin (admins : Profile []) (speaker : Profile) =
+        let foundAdmin = admins |> Array.tryFind (fun admin -> admin.Forename = speaker.Forename && admin.Surname = speaker.Surname) 
+        match foundAdmin with
+        | Some _ -> true
+        | None -> false
+        
+    wrapper.Speakers
+    |> Array.filter (speakerIsAdmin wrapper.Admins)
+    |> Array.map Profile.postAndGetId
```
Comment (pallenSL) [inline TrelloMigrate/SrmImport.fs]: This says "speakers -> isAdmin -> post"
I presume this imports only admins and is named wrong?

File: TrelloMigrate/SrmImport.fs [hunk 2]
```diff
@@ -6,11 +6,20 @@ open SrmApiClient
 //Since everything in here is causing side effects, this module isn't as functional.
 //Maybe this should be written in a imperative way instead to make that obvious? 
 
-let private importAdmins (wrapper : SrmWrapper) =
-    {wrapper with Admins = Array.map Profile.postAndGetId wrapper.Admins }
+let private importAdmins (wrapper : SrmWrapper) = Array.map Profile.postAndGetId wrapper.Admins
```
Comment (pallenSL) [inline TrelloMigrate/SrmImport.fs]: Could this be:

wrapperAdmins
|> Array.map Profile.postAndGetId

i.e. be exactly the same as the code for speakers (assuming that we are doing the same thing for admins)

File: TrelloMigrate/SrmImport.fs (hunk 1 above)
Comment (jebbinSCL) [inline TrelloMigrate/SrmImport.fs]: I messed up my logic. I'll correct it now.

File: TrelloMigrate/SrmImport.fs (hunk 2 above)
Comment (jebbinSCL) [inline TrelloMigrate/SrmImport.fs]: Changed

File: TrelloMigrate/SrmImport.fs (hunk 1 above)
Comment (jebbinSCL) [inline TrelloMigrate/SrmImport.fs]: Changed.

**Comment by pallenSL** [comment]
:shipit:

**Pull request merged**